src/client/app/translations/data.ts

```diff
@@ -1071,91 +1072,92 @@ const LocaleTranslationData = {
 		"days": "Días",
 		"decreasing": "decreciente",
-		"default.area.normalize": "Normalizar lecturas de área predeterminadas",
-		"default.area.unit": "Unidad de Área Predeterminada",
-		"default.bar.stacking": "Apilamiento de barras predeterminado",
+		"default.area.normalize": "Normalizar lecturas según el área por defecto",
+		"default.area.unit": "Unidad de área predeterminada",
+		"default.bar.stacking": "Apilar barras por defecto",
 		"default.graph.type": "Tipo de gráfico por defecto",
 		"default.graph.settings": "Configuraciones predeterminadas del gráfico",
 		"defaultGraphicUnit": "Unidad del gráfico predeterminada:",
 		"default.language": "Idioma predeterminado",
-		"default.meter.reading.frequency": "Frecuencia de la lectura del medidor predeterminada",
-		"default.site.title": "Título del Sitio Predeterminado",
-		"default.warning.file.size": "Advertencia Predeterminada del Tamaño de Archivo",
-		"default.file.size.limit": "El límite del tamaño del archivo predeterminado",
+		"default.meter.reading.frequency": "Frecuencia de lectura del medidor predeterminada",
+		"default.site.title": "Título predeterminado de la página ",
+		"default.warning.file.size": "Advertencia predeterminada de tamaño del archivo",
+		"default.file.size.limit": "Límite predeterminado de tamaño del archivo",
 		"default.help.url": "URL Ayuda",
-		"default.time.zone": "Zona del horario predeterminada",
-		"default.meter.minimum.value": "Default meter minimum reading value check\u{26A1}",
-		"default.meter.maximum.value": "Default meter maximum reading value check\u{26A1}",
-		"default.meter.minimum.date": "Default meter minimum reading date check\u{26A1}",
-		"default.meter.maximum.date": "Default meter maximum reading date check\u{26A1}",
-		"default.meter.reading.gap": "Default meter reading gap\u{26A1}",
-		"default.meter.maximum.errors": "Default maximum number of errors in meter reading\u{26A1}",
-		"default.meter.disable.checks": "Default meter disable checks\u{26A1}",
-		"delete.group": "Eliminar grupo",
+		"default.time.zone": "Zona de horario predeterminada",
+		"default.meter.minimum.value": "Revisión del valor de lectura mínima del medidor predeterminado",
+		"default.meter.maximum.value": "Revisión del valor de lectura máxima del medidor predeterminado",
+		"default.meter.minimum.date": "Revisión de la fecha de lectura mínima del medidor predeterminado",
+		"default.meter.maximum.date": "Revisión de la fecha de lectura máxima del medidor predeterminado",
+		"default.meter.reading.gap": "Deistancia predeterminada entre lecturas del medidor",
+		"default.meter.maximum.errors": "Número máximo de errores en la lectura del medidor",
+		"default.meter.disable.checks": "Desactivar revisiones de medidor predeterminado",
+		"delete.group": "Borrar grupo",
 		"delete.map": "Borrar mapa",
 		"delete.user": "Borrar usario",
 		"descending": "Descendente",
-		"discard.changes": "Descartar los Cambios",
+		"discard.changes": "Descartar los cambios",
```

Comment:
In English the words are all capitalized. Should they be here? This applies in several of the changes.

Reply:
No, Spanish conventionally uses much less capitalization than in English. In fact, despite being proper nouns, the names of languages like "English" and "Spanish" are always inglés and español, never capitalized. The same is true of days of the week and months. For book and movie titles, only the first letter is capitalized. See https://es.wikipedia.org/wiki/Harry_Potter_y_la_c%C3%A1mara_secreta as an example or https://www.thoughtco.com/capitalization-in-spanish-3080293 for more information.

Reply:
Sounds. OED capitalized for emphasis but I think following standard Spanish is good so leave as is.